examples/nodejs/src/index.js

```
@@ -35,74 +42,214 @@ async function main() {
-                    root: true,
-                    fuzzy: true,
-                },
-                {
-                    page: 1,
-                    limit: 10,
-                },
-            );
+        // First search — cache MISS (computed fresh)
+        const t1 = performance.now();
+        const first = search('الله', quranData, morphologyMap, wordMap, { lemma: true, root: true }, { page: 1, limit: 20 }, cache);
+        const d1 = (performance.now() - t1).toFixed(2);
```

Biome (2.4.4)

[error] This variable is used before its declaration.

(lint/correctness/noInvalidUseBeforeDeclaration)